src/multi-selector.js

```diff
@@ -237,7 +242,8 @@ const MultiSelector = React.createClass({
 		}
 
 		if (e && e.currentTarget) {
-			e.currentTarget.value = "";
+			e.currentTarget.value = '';
```

Comment:
@dckesler yeah we do. `this.inputChange` doesn't actually change the input, it just handles the change. Setting the value this way doesn't actually fire the change event (otherwise this case would be handled in the `onChange`), so that's why I had to add this.